spec/board_spec.rb

```diff
@@ -1,71 +1,81 @@
 require 'spec_helper'
+require 'board_helper'
 require 'board'
 require 'marks'
 
 RSpec.describe Board do
-  let(:board) {Board.new}
+  let(:board) { Board.new }
+  let(:board_helper) { BoardHelper.new(board) }
 
-  it "should return all the position" do
-    expect(board.board).to eq([[0, 1, 2], [3, 4, 5], [6, 7, 8]])
-  end
-
-  it "should be free position when it never been set" do
+  it "the position 0 is free" do
     position = 0
+    board_helper.string_to_board("   ,   ,   ")
     expect(board.free_positions.include?(position)).to be true
   end
 
-  it "should shouldn't get the spot after it set" do
+  it "the position 0 isn't free" do
```

Comment:
When just reading the description (without looking at the code) this it makes me wonder why a board does never have a free 0th position.